### ISSUE TITLE
Pretty print `/server/stats`

### ISSUE THREAD

**Issue by KellerFuchs** [hashbang/hashbang.sh#60]
Currently, the contents of `/server/stats` is a one-line JSON dump:

```
% curl https://hashbang.sh/server/stats
{"ny1.hashbang.sh": {"ip": "45.58.35.111", "location": "New York City, New York, USA", "currentUsers": 409, "maxUsers": "500"}, "to1.hashbang.sh": {"ip": "104.245.37.138", "location": "Toronto, Ontario, Canada", "currentUsers": 12, "maxUsers": "500"}, "sf1.hashbang.sh": {"ip": "45.58.38.224", "location": "San Francisco, California, USA", "currentUsers": 22, "maxUsers": "500"}, "da1.hashbang.sh": {"ip": "104.245.35.240", "location": "Dallas, Texas, USA", "currentUsers": 9, "maxUsers": "500"}}
```

It would be really helpful to output pretty printed JSON:

```
% curl https://hashbang.sh/server/stats | jq .
{
  "ny1.hashbang.sh": {
    "ip": "45.58.35.111",
    "location": "New York City, New York, USA",
    "currentUsers": 409,
    "maxUsers": "500"
  },
  "to1.hashbang.sh": {
    "ip": "104.245.37.138",
    "location": "Toronto, Ontario, Canada",
    "currentUsers": 12,
    "maxUsers": "500"
  },
  "sf1.hashbang.sh": {
    "ip": "45.58.38.224",
    "location": "San Francisco, California, USA",
    "currentUsers": 22,
    "maxUsers": "500"
  },
  "da1.hashbang.sh": {
    "ip": "104.245.35.240",
    "location": "Dallas, Texas, USA",
    "currentUsers": 9,
    "maxUsers": "500"
  }
}
```

It seems Flask's `jsonify` function can do this, but I don't have a clue about Flask (yet)

**Comment by ChickenNuggers** [comment]
`jsonify()` is a wrapped of `json.dumps()`. I can't view or test anything right now because I'm in school, but the following code should be an example of how to properly pretty-print this code:

```
import json
print(json.dumps({
  "ny1.hashbang.sh": {
    "ip": "45.58.35.111",
    "location": "New York City, New York, USA",
    "currentUsers": 409,
    "maxUsers": "500"
  },
  "to1.hashbang.sh": {
    "ip": "104.245.37.138",
    "location": "Toronto, Ontario, Canada",
    "currentUsers": 12,
    "maxUsers": "500"
  },
  "sf1.hashbang.sh": {
    "ip": "45.58.38.224",
    "location": "San Francisco, California, USA",
    "currentUsers": 22,
    "maxUsers": "500"
  },
  "da1.hashbang.sh": {
    "ip": "104.245.35.240",
    "location": "Dallas, Texas, USA",
    "currentUsers": 9,
    "maxUsers": "500"
  }
}), indent = 4, sort_keys = False)
```

**Comment by ChickenNuggers** [comment]
@KellerFuchs bump, mind taking a few mins to implement?

**Comment by KellerFuchs** [comment]
I looked at it last week, but couldn't figure out where Flask's automagic was happening and/or how to put the call to `jsonify

**Comment by daurnimator** [comment]
:+1:

**Comment by daurnimator** [comment]
https://github.com/flask-restful/flask-restful/blob/1b054a7b159da72488d199498f37f14db75caa1c/flask_restful/representations/json.py#L9

**Comment by KellerFuchs** [comment]
Seems I closed too early, this PR didn't solve the issue.

**Comment by gmcclins** [comment]
jq makes it look pretty 

`curl https://hashbang.sh/server/stats | jq '.'`

**Comment by KellerFuchs** [comment]
OK, seems it was solved with the upgrades in #107
